inference-engine/src/transformations/src/transformations/serialize.cpp

```diff
@@ -409,7 +431,7 @@ class XmlSerializer : public ngraph::AttributeVisitor {
     void on_adapter(
         const std::string& name,
         ngraph::ValueAccessor<std::shared_ptr<Function>>& adapter) override {
-        if (name == "body") {
+        if (name == "body" || name == "then_body" || name == "else_body") {
```

Comment:
Why should we hardcode some names here?

Reply:
@evolosen can we use ngfunction_2_irv10(...) with additional 'name' argument instead of this condition?

Reply:
@ilyachur we discussed this point with @evolosen, unfortunately it looks like we should hardcode these names.
We support only these names in IR and if we get here another name, it's an issue, we throw exception in this case.
Also there is a difference between processing outer network and TI/Loop/If body: we need to remove some attributes and process `m_xml_node.parent`, rather than `m_xml_node` as in case of outer network

Reply:
But we already have a place with this names - `body_names` which is specified void adapter.
Can we reuse it?

Reply:
there is a bit tricky logic in void adapter, it's not just name comparison
I can move reserved names ("body", "then_body" and so on) in some common place, 
e.g. static constexpr char[] then_body_name = "then_body", but it looks useless

Besides that model.cpp and serialize.cpp are in different libraries, I can only implement public (OPENVINO_API) helper func in some shared utils file.

Reply:
at least can we use something like that:
```
std::unordered_set<std::string> body_names = {"body", "then_body", "else_body"};
if (body_names.count(name)) ...
```

Reply:
no big difference in these solutions